python/cog/server/webhook.py

```diff
@@ -56,8 +59,11 @@ def caller(response: Any) -> None:
                 # For terminal updates, retry persistently
                 retry_session.post(webhook, json=response)
             else:
-                # For other requests, don't retry
-                default_session.post(webhook, json=response)
+                # For other requests, don't retry, and ignore any errors
+                try:
+                    default_session.post(webhook, json=response)
+                except Exception:
```

Comment:
Do we really want to catch all exceptions, not explicitly the connection errors?

Reply:
I think yes – sending this webhook is not critical to the whole thing. The main thing we're guarding against here are temporary issues, where really any error can be ignored as long as it's not our final webhook.

There's an argument that in the future we might want to be more selective here (eg, fail the prediction early if we get a 413 or something, on the basis that it won't recover during the prediction run and we'd be better off just giving up now). But I think that's something to explicitly fail on, on a case by case basis.

Reply:
Yup, I think this should probably be catching `requests.exceptions.RequestException`.